docs/antispam.rst

```diff
@@ -129,8 +129,8 @@ The following steps have to be taken to configure an additional symbol (rule) th
 
    .. code-block:: bash
 
-     docker compose scale antispam=0
-     docker compose scale antispam=1
+     docker compose up -d --scale antispam=0
+     docker compose up -d --scale antispam=1
```

Comment:
The new command to use is:
``docker compose up antispam --force-recreate -d``

Reply:
Both work, I followed the existing docs. Pick whichever.